### DISCUSSION TITLE
How to change the name of the sample app in a monorepo

### DISCUSSION THREAD

**Discussion by salieflewis** [vercel/turborepo#5691] [Help] [answered]
If I have a monorepo built with turbo, it'll typically have an `app` dir and a `packages` dir (if following a template).

Inside that `app` dir will typically be a Next app, and the directory is called `next`.

When I push up this project to vercel, and I've changed the name of this directory to `something-else`, vercel will yell at me because it is expecting a directory named `next`.

Please help, where can I change this configuration/expectation?

**Comment by kibolho** [comment, marked as answer]
@salieflewis You will change the root directory in the settings page
![Screenshot 2023-08-11 at 21 15 10](https://github.com/vercel/turbo/assets/1197923/615134a0-4791-4e58-9f24-612bacb576ba)

**Reply by salieflewis** [reply]
I may have overlooked this, thank you.
